chart/scripts/bootstrap/bootstrap.sh

```diff
@@ -180,12 +180,14 @@ function create_default_resources() {
   print_info "Add client role: realm-management:realm-admin -> $PH_USER"
   kc_user_add_client_role $KC_REALM $PH_USER realm-management realm-admin
 
-  instances='cpu-1 cpu-2 gpu-1 gpu-2'
+  instances="$(cat /instancetypes/list.txt)"
   for instance in $instances; do
     kc_role_create ROLE_IT $KC_REALM "it:${instance}"
     print_info "Bind instancetypes: it:${instance} -> everyone"
     kc_group_add_realm_role $KC_REALM everyone "it:${instance}" || true
   done
+  print_info "Add CRDs from /instancetypes/crds.yaml"
```

Comment:
Should we display the instance type's name like this?
```
print_info "Add InstanceTypes ${instances}"
```

Reply:
fix it :D